pkg/apis/config/v1alpha1/types.go

```diff
@@ -55,3 +58,13 @@ type ACME struct {
 	// +optional
 	PropagationTimeout *metav1.Duration `json:"propagationTimeout,omitempty"`
 }
+
+// Custom contains optional custom configuration for the cert-controller-manager deployment
+type Custom struct {
```

Comment:
What do you think about reusing the `ACME` struct here? `Custom` sounds very generic and the fields below are actually ACME related.

Reply:
`ACME`struct is reused as proposed.